Cargo.toml

```diff
@@ -20,7 +20,7 @@ bitflags = "2.3"
 widestring = "1"
 
 [target.'cfg(windows)'.dependencies.windows-sys]
-version = "0.52.0"
+version = "0.59.0"
 features = [
     "Win32_Foundation",
     "Win32_Security",
```

Comment:
Is the security feature maybe redundant now, or is it used by something else? I was wondering since we stopped using the `Security` module for `SC_HANDLE` now.

Reply:
`SC_HANDLE` moved from `Win32::Security` to `Win32::System::Services`. I've tried to remove `Win32::Security` and the crate builds fine, so I've committed the change.